src/services/services.ts

```diff
@@ -2111,7 +2113,7 @@ namespace ts {
                         node.parent.kind === SyntaxKind.ExternalModuleReference ||
                         isArgumentOfElementAccessExpression(node) ||
                         isLiteralComputedPropertyDeclarationName(node)) {
-                        setNameTable((<LiteralExpression>node).text, node);
+                        setNameTable((<LiteralExpression>node).text as EscapedIdentifier, node); // Literal expressions are escaped

```

Comment:
Correct! When I started it did (as it does right now), and that leaked escaped strings to wholly too many places. This probably surfaces as a bug.